src/main/java/org/sigmah/server/handler/UpdateProjectTeamMembersHandler.java

```diff
@@ -0,0 +1,102 @@
+package org.sigmah.server.handler;
+
+/*
+ * #%L
+ * Sigmah
+ * %%
+ * Copyright (C) 2010 - 2016 URD
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import com.google.inject.persist.Transactional;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.inject.Inject;
```

Comment:
Well, technically this is OK but I would favor consistency all over the code base, could you replace (multiple occurrences in the PR)

```java
import javax.inject.Inject;
```

with

```java
import com.google.inject.Inject;
```

which have 285 occurrences so far. Or maybe we could migrate everything to `javax.inject`, @Raphcal any feedback about that?